### PR TITLE
Standardize help texts for commands and arguments

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: croud/__main__.py [hunk 1]
```diff
@@ -290,7 +293,7 @@
                         "calls": org_users_add,
                     },
                     "remove": {
-                        "help": "Remove user from organization",
+                        "help": "Add the selected user from an organization.",
```
Comment (lukas-bernert) [inline croud/__main__.py]: "to" instead of "from"?

File: croud/__main__.py [hunk 2]
```diff
@@ -307,8 +310,7 @@
         "help": "Manage CrateDB Cloud users.",
         "sub_commands": {
             "list": {
-                "help": "List all users within organizations that the "
-                "logged in user is part of.",
+                "help": "List all users within the given organizations.",
```
Comment (lukas-bernert) [inline croud/__main__.py]: sometimes we use "specified" and sometimes "given". Maybe we should agree on one?

File: croud/__main__.py [hunk 3]
```diff
@@ -319,7 +321,7 @@
                 "help": "Manage CrateDB Cloud user roles.",
                 "sub_commands": {
                     "add": {
-                        "help": "Adds a role to a user.",
+                        "help": "Grant the selected user to the chosen role.",
```
Comment (lukas-bernert) [inline croud/__main__.py]: same here. now we have specified, given, chosen

File: croud/__main__.py [hunk 4]
```diff
@@ -335,7 +337,7 @@
                         "calls": roles_add,
                     },
                     "remove": {
-                        "help": "Removes a role from a user.",
+                        "help": "Revoke the chosen role from the selected user.",
```
Comment (lukas-bernert) [inline croud/__main__.py]: Can we still call it "remove" instead of revoke? as the command is called "remove" as well?

File: croud/cmd.py [hunk 5]
```diff
@@ -408,7 +428,10 @@ def consumer_eventhub_lease_storage_connection_string_arg(
     group.add_argument(
         "--consumer-eventhub-lease-storage-connection-string",
         type=str,
-        help="Connection string of the lease storage for the EventHub consumer.",
+        help=(
+            "The connection string to a lease storage for the Azure EventHub "
+            "consumer write its lease files to."
```
Comment (lukas-bernert) [inline croud/cmd.py]: ...lease storage **where?** the Azure EventHub consumer write**s** it's lease files to.. ?

File: croud/cmd.py [hunk 6]
```diff
@@ -432,7 +458,7 @@ def consumer_schema_arg(
     group.add_argument(
         "--consumer-schema",
         type=str,
-        help="Database schema in which to insert.",
+        help="The CrateDB database schema used by the Azure EventHub consumer.",
```
Comment (lukas-bernert) [inline croud/cmd.py]: above we say "...to use."
and here we say "used by...."

can we stick to one?

File: croud/__main__.py [hunk 7]
```diff
@@ -192,16 +191,18 @@
                 "calls": project_create,
             },
             "list": {
-                "help": "Lists all projects for the current "
-                "user in the specified region.",
+                "help": (
+                    "Lists all projects the current user has access to in "
```
Comment (lukas-bernert) [inline croud/__main__.py]: Sometimes we use "Lists" sometimes "List". Can we please stick to "List"?

File: croud/__main__.py (hunk 4 above)
Comment (chaudum) [inline croud/__main__.py]: Same here:
```suggestion
                        "help": "Unassign a role from the selected user.",
```

File: croud/cmd.py [hunk 8]
```diff
@@ -208,7 +208,7 @@ def region_arg(req_args: _ArgumentGroup, opt_args: _ArgumentGroup) -> None:
         "--region",
         choices=["westeurope.azure", "eastus.azure", "eastus2.azure", "bregenz.a1"],
         type=str,
-        help="Switch region that command will be run on.",
+        help="Switches the region that command will be run on.",
```
Comment (chaudum) [inline croud/cmd.py]: It does not switch in a sense that it is permanent.
It uses the region for a specific command.

File: croud/__main__.py (hunk 2 above)
Comment (MarkusH) [reply croud/__main__.py]: I went with "specified".

File: croud/__main__.py (hunk 3 above)
Comment (MarkusH) [reply croud/__main__.py]: Well, "Add the specified role to the specified user" sounds really odd in my ears. Any suggestion?

File: croud/cmd.py (hunk 5 above)
Comment (MarkusH) [reply croud/cmd.py]: I hope the rephrasing makes it more clear.

File: croud/__main__.py (hunk 7 above)
Comment (MarkusH) [reply croud/__main__.py]: Oops. Completely missed that one :see_no_evil:

File: croud/__main__.py [hunk 9]
```diff
@@ -124,10 +124,10 @@
         },
     },
     "consumer-sets": {
-        "help": "Manage consumer sets for CrateDB Cloud for Azure IoT products.",
+        "help": "Manage consumer sets in CrateDB Cloud.",
```
[the remark this answers is not in the document]
Comment (MarkusH) [reply croud/__main__.py]: I dropped "CrateDB Cloud" from all commands except login and logout as well as the help for the top-level command.

File: croud/__main__.py (hunk 3 above)
Comment (lukas-bernert) [reply croud/__main__.py]: No but what about "Grant the selected user the specified role?"

**Pull request merged**